### ISSUE TITLE
Mask-RCNN

### ISSUE THREAD

**Issue by ailiwei17** [ailiwei17/note#1]
# 环境
Ubuntu16.04
Driver Version: 430.64 
CUDA:cuda_10.0.130_410.48_linux
cudnn:cudnn-10.0-linux-x64-v7.6.5.32
TensorFlow 1.14.0 + Keras 2.2.5
## NVDIA 显卡驱动安装 
参考网址：[https://blog.csdn.net/u013289254/article/details/103499483](url)
跳过禁用X服务，否则会导致黑屏
## 环境配置
参考网址：[https://blog.csdn.net/weixin_40539826/article/details/109952914](url)
# 运行
参考网址：[https://blog.csdn.net/qq_27399933/article/details/107037661?utm_medium=distribute.pc_relevant.none-task-blog-baidujs_utm_term-8&spm=1001.2101.3001.4242](url)
遇到Jupyter notebook 报错500 : Internal Server Error，确认Jupyter notebook 是否安装在规定的虚拟环境，不要直接在Jupyter notebook 中添加python内核，卸载重装
## 在pycharm中运行
注释掉`get_ipython().run_line_magic('matplotlib', 'inline')`
忽略自定义库的报错即可以运行
